lms/djangoapps/discussion/templates/discussion/user_profile.html

```diff
@@ -0,0 +1,62 @@
+## mako
+
+<%! main_css = "style-discussion-main" %>
+
+<%page expression_filter="h"/>
+<%inherit file="../main.html" />
+<%namespace name='static' file='../static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+from django.template.defaultfilters import escapejs
+from openedx.core.djangolib.js_utils import (
+    dump_js_escaped_json, js_escaped_string
+)
+%>
+
+<%block name="bodyclass">discussion-user-profile</%block>
+<%block name="pagetitle">${_("Discussion - {course_number}").format(course_number=course.display_number_with_default)}</%block>
+
+<%block name="headextra">
+<%include file="_js_head_dependencies.html" />
+</%block>
+
+<%block name="js_extra">
+<%include file="_js_body_dependencies.html" />
+<%static:js group='discussion'/>
+<script type="text/javascript">
+    RequireJS.define('DiscussionUserProfileView', [], function() {return window['DiscussionUserProfileView'];});
+</script>
+
+<%static:require_module module_name="discussion/js/discussion_profile_page_factory" class_name="DiscussionProfilePageFactory">
+    DiscussionProfilePageFactory({
+        courseId: '${unicode(course.id) | n, js_escaped_string}',
+        el: $(".discussion-user-threads")
+    });
+</%static:require_module>
+</%block>
+
+<%include file="../courseware/course_navigation.html" args="active_page='discussion'" />
+
+<section class="container">
```

Comment:
We need a heading here, as the direct child of this `section`. Use whatever is next in the heading hierarchy (probably an H2).